openvpn/installpkg/recipes/copy_file.rb

```diff
@@ -0,0 +1,7 @@
+cookbook_file "Copy a file" do  
```

Comment:
Why do we need this?

Reply:
I use this cookbook to upload this file for use in serverspec-init. But, with "kitchen verify", I see this is unnecessary. I will delete these files and update "README.md"